components/overflow-group/overflow-group-mixin.js

```diff
@@ -311,7 +320,9 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 
 		this._updateOverflowItemsRequested = true;
 		setTimeout(() => {
-			this._overflowItems = this._slotItems.map(node => this.convertToOverflowItem(node));
+			this._itemLayouts = this._getItemLayouts(this._slotItems);
+			this._overflowItems = this._slotItems.map((node) => this.convertToOverflowItem(node));
+			this._chomp();
```

Comment:
When selection status changes, the width of an item can also change and that can affect chomping. I put this here in the mixin since in theory mutations to other components using the mixin could also cause layout changes.